cookbooks/OpenAI-ChatCompletion-AIConfigWrapper/openai_wrapper.ipynb

```diff
@@ -2,34 +2,24 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install python-aiconfig\n",
```

Comment:
You might need to pull latest since this seems to revert my ipynb changes I just pushed to main